### PR TITLE
completed task 9

### PR DESCRIPTION
**Pull request by nearhanfarhan**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -321,4 +320,23 @@ describe("far-han-news tests", () => {
...
+      return request(app).delete(`/api/comments/${comment_id}`).expect(404).then(({body})=> {
+        expect(body.msg).toBe("Comment not found")
+      })
+    })
+    test("400 DELETE: should return an error when passed a comment ID of invalid type", () => {
+      const comment_id = "bananas"
+      return request(app).delete(`/api/comments/${comment_id}`).expect(400).then(({body})=> {
+        expect(body.msg).toBe("Bad request")
+      })
+
```
Comment (Rayhaan2k9) [inline __tests__/app.test.js]: Great work on these tests! 😄

File: models/comments.models.js [hunk 2]
```diff
@@ -17,3 +17,14 @@ exports.insertCommentsByArticle = (article_id, author, body) => {
     return result.rows[0];
   });
 };
+
+exports.removeCommentById = (comment_id) => {
+  text = "DELETE FROM comments WHERE comment_id = $1 RETURNING*"
+  params = [comment_id]
```
Comment (Rayhaan2k9) [inline models/comments.models.js:23]: I know that these variables work, but without const or let, they are global scoped and we don't want that! Please put ```const``` before them!

File: models/comments.models.js [hunk 3]
```diff
@@ -17,3 +17,14 @@ exports.insertCommentsByArticle = (article_id, author, body) => {
     return result.rows[0];
   });
 };
+
+exports.removeCommentById = (comment_id) => {
+  text = "DELETE FROM comments WHERE comment_id = $1 RETURNING*"
```
Comment (Rayhaan2k9) [inline models/comments.models.js:22]: There is no need for ```RETURNING *``` for this one as we can't return anything from a delete request

**Review by Rayhaan2k9 — APPROVED** [review]
Great work, Farhan! Left a couple comments for you. Please fix these, pus and then merge it 🙂

File: controllers/users.controllers.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+const { fetchAllUsers } = require("../models/users.models")
+
+exports.getAllUsers = (request, response, next) => {
+    console.log("in controller")
```
Comment (EddTheDuck) [inline controllers/users.controllers.js:4]: Remove console.logs before pushing :)

**Review by EddTheDuck — COMMENTED** [review]
Looks good 👍

**Pull request merged**